experimenter/experimenter/experiments/constants.py

```diff
@@ -19,6 +19,7 @@ class Channel(models.TextChoices):
     DEVELOPER = "developer"
     STAGING = "staging"
     PRODUCTION = "production"
+    VPN = "vpn"
```

Comment:
Wait why do we need another channel here?

Reply:
oh, you remember we discussed that we don't need to create a channel, as the VPN stage will always point to Remote stage and the Prod will always point to the prod remote, so we don't need to divide the channel essentially.

Reply:
We can just use single channel, this is still pending to update for Monitor, but for VPN- starting with the single channel

Reply:
No but I mean why can't we just use one of the existing channels like `production`? Why do we need to make a new one just for VPN?

Reply:
oh, I see what you mean, because I was thinking that they can get confused with `production` as a channel option on `stage experimenter console`, what do you think @jaredlockhart?

Reply:
No I'd say just leave it as 'production'.  We should try to avoid blowing up the list of channels as much as possible I think.  If people want to change it later we can but I think 'production' still has meaning here.  And I mean that's exactly what we did for monitor too.

Reply:
for monitor they have two options- stage and production